foodx_devops_tools/file_maintainer_entry.py

```diff
@@ -57,13 +57,35 @@ def _get_filesystem_capacity_used(directory: pathlib.Path) -> int:
 
     capacity_used_percent = int((1 - (available_bytes / size_bytes)) * 100)
 
-    log.info("filesystem size (bytes), {0}".format(size_bytes))
-    log.info("filesystem available size (bytes), {0}".format(size_bytes))
+    log.info("filesystem size (bytes): {0}".format(size_bytes))
```

Comment:
stick with the original comma format. colons are often used as field separators in log lines so using colons inside a field makes parsing fields later much more challenging.